aea/cli/init.py

```diff
@@ -37,31 +37,31 @@
 @click.option("--author", type=str, required=False)
 @click.option("--reset", is_flag=True, help="To reset the initialization.")
 @click.option("--local", is_flag=True, help="For init AEA locally.")
-@click.option("--subscribe", is_flag=True, help="For developers subscription.")
+@click.option("--no_subscribe", is_flag=True, help="For developers subscription.")
```

Comment:
use `--no-subscribe` instead? It's easier to type, and in general, I see most of the flags use only hyphens `-` and not underscores `_`